test/stdlib/symbol-visibility.test-sh

```diff
@@ -0,0 +1,3 @@
+// RUN: %llvm-nm --defined-only --extern-only --demangle %platform-dylib-dir/%target-library-name(swiftCore) > %t/all.txt
```

Comment:
Would it be possible to test also `swiftRemoteMirror` and possibly other libraries that use LLVM support (possibly `swiftReflection`)?

Reply:
Sure, we can add the other two DSOs.  I’ll do that in a little while.   Sounds like a good idea to do that :)